### PR TITLE
Update ConnectingExpertiseInvoicing.md

### PR DESCRIPTION
**Pull request by robeecaneda1**
Added additional steps on how to process timesheets on Connecting-Expertise

### CODE REVIEW

File: Bookkeeping/ConnectingExpertiseInvoicing.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+# Connecting Expertise
 ====================
 
 Login: https://customer.connecting-expertise.com/index.php?event=contractor.login
 
 Each contractor has his own login account.  
-Fill in the timesheet and Save + Activate.  
 
-<!-- TODO: Fill in next steps once the timesheet has been approved -->
+- Login as employee> Timesheets> Select correct payroll month> Timesheets> Summary> Type: Normal Days> Select date> Save> Activat
```
Comment (Laoujin) [inline Bookkeeping/ConnectingExpertiseInvoicing.md]: Typo "activate"?

File: Bookkeeping/ConnectingExpertiseInvoicing.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+- Take a screenshot of the saved Connecting-Expertise timesheet and save on the monthly invoicing folder.
+
+**Note**
+
+When processing timesheets, make sure to use the correct employee login. Use the timesheet submitted by employee to enter correct information. If no descripancies, after "Save", hit "Activate".
```
Comment (Laoujin) [inline Bookkeeping/ConnectingExpertiseInvoicing.md]: discrepancies

File: Bookkeeping/ConnectingExpertiseInvoicing.md (hunk 1 above)
Comment (robeecaneda1) [reply Bookkeeping/ConnectingExpertiseInvoicing.md]: Done correcting the typos. I forgot that github doesn't have auto correct.

File: Bookkeeping/ConnectingExpertiseInvoicing.md (hunk 2 above)
Comment (robeecaneda1) [reply Bookkeeping/ConnectingExpertiseInvoicing.md]: Done correcting the typos. I forgot that github doesn't have auto correct.

**Pull request merged**